jina/docker/hubio.py

```diff
@@ -256,6 +256,18 @@ def build(self) -> Dict:
                         is_push_success = True
                     except Exception:
                         self.logger.error(f'can not push to the registry')
+                
+                if self.args.host_info:
+                    info, env_info = get_full_version()
+                    _host_info = {
+                        'jina': info,
+                        'jina_envs': env_info,
+                        'docker': self._raw_client.info(),
+                        'build_args': vars(self.args)
+                    }
+                else:
+                    _host_info = ''
+                    self.logger.debug('Skipping writing host_info to database')
```

Comment:
`_host_info =  ''` should be added to L188 to ensure this var is always `defined` regardless `is_build_succcess`

Reply:
This is fixed.